cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -221,7 +222,7 @@ func coreDNSAddon(cfg *kubeadmapi.ClusterConfiguration, client clientset.Interfa
 func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, client clientset.Interface) error {
 	coreDNSConfigMap := &v1.ConfigMap{}
 	if err := kuberuntime.DecodeInto(clientsetscheme.Codecs.UniversalDecoder(), configBytes, coreDNSConfigMap); err != nil {
-		return errors.Wrap(err, "unable to decode CoreDNS configmap")
+		return errors.Wrap(err, UnabledeCodeCoreDNS+"configmap")
```

Comment:
there is a typo in the constant name, but also please use:
`errors.Wrapf(err, "%s ConfigMap", UnableToDecodeCoreDNS)`

please uppercase all of these:
configmap -> ConfigMaps
clusterroles -> ClusterRole
clusterolebindings -> ClusterRoleBinding
serviceaccount -> ServiceAccount
deployment -> Deployment